tests/manifests_and_classes_spec.rb

```diff
@@ -1,84 +1,78 @@
-describe "Task 1:" do
-  it 'Define the cowsayings::cowsay class' do
-    file("#{MODULE_PATH}cowsayings/manifests/cowsay.pp")
-      .should be_file
-    file("#{MODULE_PATH}cowsayings/manifests/cowsay.pp")
+describe "Task 1:", host: :localhost do
+  it 'Display the module path' do
+    file('/root/.bash_history')
       .content
-      .should match /class\s+cowsayings::cowsay\s*{/
+      .should match /puppet config print modulepath/
   end
 end
 
-describe "Task 2:" do
-  it 'Include the cowsayings::cowsay class in a test manifest' do
-    file("#{MODULE_PATH}cowsayings/examples/cowsay.pp")
-      .should be_file
-    file("#{MODULE_PATH}cowsayings/examples/cowsay.pp")
+describe "Task 2:", host: :localhost do
+  it 'Make the cowsay module directory' do
+    file('/root/.bash_history')
       .content
-      .should match /include\s+cowsayings::cowsay/
+      .should match /mkdir \-p cowsay\/manifests/
   end
 end
 
-describe "Task 3:" do
-  it 'Apply the test manifest to install the cowsay package' do
-    file('/usr/local/bin/cowsay')
+describe "Task 3:", host: :localhost do
+  it 'Create the cowsay default manifest' do
+    file('/etc/puppetlabs/code/environments/production/modules/cowsay/manifests/init.pp')
```

Comment:
This is getting to where using rspec-puppet to compile the class would be helpful. But I'm not sure how to do that yet.